packages/v4/src/content/design-guidelines/components/number input/number-input.md

```diff
@@ -0,0 +1,58 @@
+---
+id: Number input
+section: components
+---
+
+A **number input** provides a quick and effective way for users to enter and modify a numeric value. Similar to input groups, number inputs combine two input types into one control: A text input field with buttons on either side. When combined, these elements allow users to input numeric values in a responsive and scalable way.
+
+## Elements
+<img src="./img/number_input_anatomy.png" alt="Number input elements" width="50%"/>
```

Comment:
I feel like your images are all scaling differently. Some of them seem too large. I think it's because you are using `width="50%"`. I think our standard is to use the absolute pixel width of the original artboard from Sketch. @mmenestr can you confirm?

Reply:
@redallen confirmed, attributes have been updated. @mcarrano do you mind reviewing this again?